docs/dusd-spec.tex

```diff
@@ -472,17 +472,26 @@ \subsection{Price module}
 
 Price module:
 \begin{itemize}
-  \item The on-chain code will be written in Plutarch
-  \item The off-chain code will be built as a PAB
-  \item The datum will contain a map from timestamps (in POSIX time) to price
-    values
-  \item The script address will be dependent on three admin public
-    keys\footnote{
-      These admin keys can only be updated when the protocol is updated.
-      This decision will be enforced by using the keys to determine the script
-      address, rather than putting them into the datum.
-    }
-\end{itemize}
+  \item The off-chain code will be built in CTL.
+  \item The on-chain code will be written in Plutarch.
+  \item The validator will validate the following:
+    \begin{itemize}
+      \item There is a previous price oracle UTxO being spent.
+      \item The previous price oracle UTxO is being spent by Admin.
+      \item The previous price oracle UTxO contains the Oracle NFT.
+      \item The datum has a list of tuples containing USD Price of ADA and the POSIX timestamp 
```

Comment:
@Geometer1729 Do you think it would be reasonable to store a [iso8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp? I think it is more future proof than a POSIX timestamp

Reply:
The question is how feasible that is for the on-chain code

Reply:
The time stamps used in the script context in the on-chain code are future proof they use true integers (onchain code can't even use Ints). They are also not quite POSIX time stamps cause they start when cardano started not 1970 but they are called that in all the docs and I don't know a better term.